src/pypyjs.js

```diff
@@ -472,18 +477,24 @@ pypyjs.prototype.addModule = function addModule(name, source) {
   });
 };
 
+
+
 // Method to execute python source directly in the VM.
 //
 // This is the basic way to push code into the pypyjs VM.
 // Calling code should not use it directly; rather we use it
 // as a primitive to build up a nicer execution API.
 //
-pypyjs.prototype._execute_source = function _execute_source(code) {
+pypyjs.prototype._execute_source = function _execute_source(code, preCode) {
   console.log('executing: ' + code);
   const Module = this._module;
   let code_ptr;
+  if (!preCode) {
```

Comment:
I don't have the linter installed yet but I'm pretty sure it won't like reassignment of function args :smile: 

`const _preCode = preCode ? preCode : '';`